SwiftCSVTests/QuotedTests.swift

```diff
@@ -33,4 +33,34 @@ class QuotedTests: XCTestCase {
             "age": "8"
         ])
     }
+
+    func testEmbeddedQuotes() throws {
```

Comment:
Thanks for adding this. It's hard to believe that  this is how CSV quotes work. Ugh. Expected escape sequences like `"\"...\""` tbh

Maybe add a link to https://www.rfc-editor.org/rfc/rfc4180.html in the helpful block comment in this test case, and potentially also in `enquoted(wihenContaining:)`?

Reply:
CSV originated 50 years ago in the IBM ecosystem, not the Bell Labs ecosystem. Popular IBM languages back then (COBOL, Pascal, Ada, FORTRAN) used character doubling rather than a distinct escape character.

Whereas, the ASCII teletypes that the Bell Labs engineers used as terminals had a distinct escape character (0x1B, often written `^]`). So that philosophy carried over into the programming languages that they invented, such as B (escape character: `*`) and C (escape character: `\`).